backend/clickhouse/traces.go

```diff
@@ -310,17 +311,23 @@ func (client *Client) ReadTracesMetrics(ctx context.Context, projectID int, para
 		}
 	}
 
+	groupStr := ""
+	for _, group := range groupBy {
+		groupStr += fmt.Sprintf("toString(TraceAttributes['%s']), ", group)
```

Comment:
I think this is vulnerable to sql injection b/c group comes directly from user input - can we hardcode `TraceAttributes['ingested']` or use `sb.Var` here?

Reply:
good catch, i actually had tested for this and wasn't able to perform an attack (clickhouse detects that multiple commands are trying to run as part of one query and fails) but i've refactored this to use `sb.Var`